### PR TITLE
Move bulk import logic into bulk creator

### PR DESCRIPTION
**Pull request by longhotsummer**
This pulls all the business logic into one place, rather than splitting it across the view and the bulk creator plugin.

### CODE REVIEW

File: indigo_app/static/javascript/indigo/views/work.js [hunk 1]
```diff
@@ -346,15 +346,15 @@
     },
   });
 
-  Indigo.BulkImportWorksView = Backbone.View.extend({
+  Indigo.BatchAddWorkView = Backbone.View.extend({
```
Comment (goose-life) [inline indigo_app/static/javascript/indigo/views/work.js]: not sure why but I'm no longer seeing a progress bar

File: indigo/bulk_creator.py [hunk 2]
```diff
@@ -167,77 +170,101 @@ def get_works(self, view, table, dry_run):
...
+                if info['status'] != 'error' and info.get('amends'):
+                    # this will check duplicate works as well
+                    # (they won't overwrite the existing works but the amendments will be linked)
+                    self.link_amendment(info['work'], info)
+
+        return works
+
+    def create_work(self, view, row, idx, dry_run):
+        # copy all row details
+        info = {k: v for k, v in row.iteritems()}
```
Comment (goose-life) [inline indigo/bulk_creator.py]:
```suggestion
        info = row
```

Comment (longhotsummer) [reply indigo/bulk_creator.py]: Heheh.

File: indigo_app/static/javascript/indigo/views/work.js (hunk 1 above)
Comment (longhotsummer) [reply indigo_app/static/javascript/indigo/views/work.js]: hard refresh?

**Pull request merged**